### DISCUSSION TITLE
msgpack support in mcap cat --json output

### DISCUSSION THREAD

**Discussion by jon-chuang** [foxglove/mcap#1099] [General]
Similar to https://github.com/foxglove/mcap/issues/763, msgpack is also a popular schemaless encoding

**Comment by defunctzombie** [comment]
@jon-chuang Please me mindful when using the bug label. The bug label is for broken functionality. If you believe this issue is a bug please explain what existing functionality that we've documented is not working.

**Comment by jtbandes** [comment]
@defunctzombie I think it's understandable as there's no way for community members to file a non-bug issue from [this page](https://github.com/foxglove/mcap/issues/new/choose) 😄

**Comment by jtbandes** [comment]
I'm going to convert this to a discussion since we haven't yet settled on the spec updates to "well-known" encodings for msgpack usage.
